crates/freeze/src/types/schemas.rs

```diff
@@ -15,6 +16,16 @@ pub struct Table {
 
     /// sort order for rows
     pub sort_columns: Option<Vec<String>>,
+
+    /// metadata about a table
+    pub meta: Option<TableMeta>,
+}
+
+/// metadata to associated with a table
+#[derive(Clone, Debug, PartialEq)]
+pub struct TableMeta {
```

Comment:
is there a reason to use `TableMeta` instead of just stashing the `log_decoder` directly inside the `Table`? seems like it would be cleaner that way

Reply:
seemed prudent to have a container type as I assumed there be extra info needed to pass through
that may not logically fit with the log_decoder but happy to remove if you think is best